modules/optimization/doc/content/modules/optimization/getting_started/InvOptTheory.md

```diff
@@ -0,0 +1,376 @@
+\begin{equation}\label{eq:adjointGrad}
+\frac{\mathrm{d}J}{\mathrm{d}\mathbf{p}} = \frac{\partial J}{\partial\mathbf{p}}+\mathbf{\lambda}\frac{\partial\mathbf{g}}{\partial\mathbf{p}} ,
+\end{equation}
+where $\partial J/\partial\mathbf{p}$ accounts for the regularization in [eq:objective] and $\mathbf{\lambda}$ is the adjoint variable solved for from the adjoint equation
+\begin{equation}\label{eq:adjoint}
+\left(\frac{\partial\mathbf{g}}{\partial\mathbf{u}}\right)^\top \mathbf{\lambda}= \left(\frac{\partial J}{\partial\mathbf{u}}\right)^\top ,
+\end{equation}
+where $\left(\partial\mathbf{g}/\partial\mathbf{u}\right)^\top$ is the adjoint of the Jacobian for the original forward problem, $\mathbf{g}$, and $\left(\partial J/\partial\mathbf{u}\right)^\top$ is a body force like term that accounts for the misfit between the computed and experimental data.  Thus, the solution to [eq:adjoint] has the same complexity as the solution to the forward problem.
+
+The remaining step for evaluating the derivative of the PDE in [eq:adjointGrad] is to compute $\partial\mathbf{g}/\partial\mathbf{p}$, the derivative of the PDE with respect to the parameter vector.  The form this term takes is dependent on the physics (e.g. mechanics or heat conduction) and the parameter being optimized (e.g. force inversion versus material inversion).  In what follows, we will derive the adjoint equation for steady state heat conduction and the gradient term for both force and material inversion.
+
+In the following, the adjoint method given in [eq:adjointGrad] and [eq:adjoint] is derived from [eq:objective] following the derivation given by [!citet](bradley2013pde).  Neglecting the regularization in [eq:objective] the total derivative of $J$ with respect to $\mathbf{p}$ using the chain rule is given by
+\begin{equation}\label{eq:objectiveGrad}
+\frac{\mathrm{d}J(\mathbf{u},\mathbf{p})}{\mathrm{d}\mathbf{p}} = \frac{\partial J}{\partial \mathbf{u}}\frac{\partial \mathbf{u}}{\partial\mathbf{p}}.
+\end{equation}
+We can solve easily compute  $\frac{\partial \mathbf{u}}{\partial\mathbf{p}}$ and so we need to rearrange  The physics constraint of the PDE from [eq:optimization], $\mathbf{g}\left(\mathbf{u},\mathbf{p}\right)=\mathbf{0}$, implies that
+\begin{equation} \label{eq:gradZero}
+	\begin{aligned}
+	\frac{\mathrm{d}\mathbf{g}(\mathbf{u},\mathbf{p})}{\mathrm{d}\mathbf{p}} & = 0  \\
+	\frac{\mathrm{d}\mathbf{g}(\mathbf{u},\mathbf{p})}{\mathrm{d}\mathbf{p}} & = \frac{\partial \mathbf{g}}{\partial \mathbf{u}}\frac{\partial \mathbf{u}}{\partial\mathbf{p}} + \frac{\partial \mathbf{g}}{\partial \mathbf{p}}= 0 \\
+\end{aligned}
+\end{equation}
+Rearranging the last part of [eq:gradZero] gives
+\begin{equation}\label{eq:gradZeroArrange}
+\frac{\partial \mathbf{u}}{\partial\mathbf{p}}=-\left(\frac{\partial g}{\partial \mathbf{u}}\right)^{-1}\frac{\partial g}{\partial \mathbf{p}}
```

Comment:
In this equation and the next, `g` needs to be bolded.

Reply:
A lot more equations needs to have \mathbf{g}.

Reply:
done